core/ivr/twiml/twiml.go

```diff
@@ -468,8 +476,13 @@ func responseForSprint(number urns.URN, resumeURL string, w flows.ActivatedWait,
 				languageCode := locale.ToISO639_2()
 
 				if _, valid := validLanguageCodes[languageCode]; !valid {
-					languageCode = ""
+					if _, valid := validLanguagesInitialCode[languageCode[:2]]; !valid {
```

Comment:
@teehamaral - I have checked the method `locale.ToISO639_2()` and in some cases, it returns a blank string. So probably, we need some guard condition to prevent a slice of `languageCode` when it is an empty string.
Everything else looks good to me.

Reply:
@ihor-palii - I've implemented on this commit https://github.com/greatnonprofits-nfp/mailroom/pull/23/commits/bd7930e267274cc9991d1530140c03a6512335de